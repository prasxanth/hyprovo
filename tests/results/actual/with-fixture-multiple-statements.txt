2021-11-24 22:51:24.498 - ERROR - Fail ✗... (test-add-variable): sum should be 2
2021-11-24 22:51:24.500 - ERROR - Fail ✗... (test-add-variable): sum should be 3
2021-11-24 22:51:24.504 - INFO - Pass ✓... (test-sub-variable): difference should be 0
2021-11-24 22:51:24.506 - ERROR - Fail ✗... (test-sub-variable): difference should be 4
