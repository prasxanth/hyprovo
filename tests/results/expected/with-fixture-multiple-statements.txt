2021-11-24 22:50:31.312 - ERROR - Fail ✗... (test-add-variable): sum should be 2
2021-11-24 22:50:31.315 - ERROR - Fail ✗... (test-add-variable): sum should be 3
2021-11-24 22:50:31.320 - INFO - Pass ✓... (test-sub-variable): difference should be 0
2021-11-24 22:50:31.322 - ERROR - Fail ✗... (test-sub-variable): difference should be 4
